.travis.yml

```diff
@@ -1,6 +1,7 @@
+before_install: gem install bundler -v '1.15.0'
```

Comment:
Probably should just be `gem install bundler`, but maybe locking it is better, dunno.